packages/drivers/local-driver/src/localDocumentDeltaConnection.ts

```diff
@@ -80,8 +81,29 @@ export class LocalDocumentDeltaConnection extends DocumentDeltaConnection {
 	/**
 	 * Submits a new signal to the server
 	 */
-	public submitSignal(message: string): void {
-		this.emitMessages("submitSignal", [[message]]);
+	public submitSignal(content: string): void {
+		this.emitMessages("submitSignal", [[content]]);
+	}
+
+	public submitSignal2(content: unknown): void {
+		// WARNING:
+		// This code here is only for demonstration purposes. While driver can do encoding here,
+		// it would need to do symmetrical decoding for "signal" event payloads, as well as this.initialSignals
+		// From efficiency POV, it does not make sense to implement this method if protocol does not support property
+		// binary payloads.
+		// There are two ways to accomplish it:
+		// 1. Leave it as is at socket.io level - it can transfer mixes (JS + binary) messages just fine. On service side,
+		//    likely would need to serialize it into binary blob and pass around binary. Maybe doing encodeJsonableOrBinary
+		//    on service side is fine (saves bandwidth as we do not waste +33% overhead of base64 encoding), but this makes
+		//    service side less efficient (compared if service does some more efficient binary serialization)
+		// 2. Serialize into binary here (in driver), and deserialize back on receiving signals, but service and socket.io
```

Comment:
Hi @vladsud, 
From above 1st comment, we also need decode on service side before sending to other clients? Means if we use  encodeJsonableOrBinary on service side then we need  decodeJsonableOrBinary on the service side itself not on client side if my understanding is correct